virtual_device.fragment

```diff
@@ -0,0 +1,31 @@
+# Drivers used by all Android virtual devices
+CONFIG_BATTERY_GOLDFISH=m
+CONFIG_BT_HCIVHCI=m
+CONFIG_BT_HCIUART=m
+CONFIG_CPUFREQ_DUMMY=m
+CONFIG_DMABUF_HEAPS_SYSTEM=m
+CONFIG_DRM_VKMS=m
+CONFIG_MAC80211_HWSIM=m
+CONFIG_RTC_DRV_TEST=m
+CONFIG_USB_DUMMY_HCD=m
+CONFIG_VIRTIO_BALLOON=m
+CONFIG_VIRTIO_PMEM=m
+CONFIG_USBIP_CORE=m
+CONFIG_USBIP_VHCI_HCD=m
+
+# Drivers to run specific sanity tests
+CONFIG_TEST_STACKINIT=m
+CONFIG_TEST_MEMINIT=m
+
+# Cuttlefish-specific drivers
+CONFIG_VIRT_WIFI=m
+
+# Auto-specific drivers
+CONFIG_BT_HCIBTUSB=m
+CONFIG_CAN_GS_USB=m
+CONFIG_WLAN_VENDOR_MEDIATEK=y
+CONFIG_MT76x0U=m
+CONFIG_MT76x2U=m
+
+# TV-specific drivers
```

Comment:
As far as I understood, this fragment will be used for building a guest kernel, so I have a question: Do we need there "# Auto-specific drivers" and "# TV-specific drivers" ? Since they are hardware specific, and we don't have a real hardware for now in guest ....

Reply:
Hi, @arminn .
Mentioned flags are removed.
Changes are tested.

Reply:
@dterletskiy plz add to patch description after your signoff:

Reviewed-by: Andrii Chepurnyi <andrii_chepurnyi@epam.com>

and I will merge it

Reply:
Hi, @arminn.
Done.